frontend/src/app/About/Contributors.tsx

```diff
@@ -0,0 +1,96 @@
+import { Globe } from 'iconoir-react';
+import { contributorStructure } from './index';
+import styles from "./Contributors.module.scss"
+
+interface ContributorsProps {
+	currContributors: contributorStructure[];
+	alumniContributors: contributorStructure[];
+}
+
+
+const Contributors: React.FC<ContributorsProps> = ({ currContributors, alumniContributors }) => {
+	/* 	console.log(currContributors)
+		console.log(alumniContributors)
+		console.log(currContributors[0]?.name)
+		console.log(currContributors[0]?.img.seriousBase64);
+	
+	 */
+
+	console.log(currContributors)
+	const base64Root = 'data:image/jpeg;base64,'
+
+	const alumniByGradYear: { [key: number]: contributorStructure[] } = {};
```

Comment:
Not super important, but you can convert this and the following `forEach` loop into a `useMemo` React hook using the `[Array.prototype.reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)` function. Memoization will prevent re-renders if they were to occur.